_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -156,120 +136,120 @@ The `input` element does not have a semantic role that is a widget role, but sti
 
 #### Failed Example 1
 
-Unknown autocomplete term.
+This `autocomplete` [attribute value][] does not have a [correct autocomplete field][].
 
 ```html
 <label>Username<input autocomplete="badterm"/></label>
 ```
 
 #### Failed Example 2
 
-Term `work` not allowed before `photo`.
+This `autocomplete` [attribute value][] has the `work` token on a [correct autocomplete field][] that does not support this.
```

Comment:
Is work not allowed with photo, or is it in the wrong order?

Reply:
It is not allowed. I'll update the text to make this clearer.